Documentation/Storage-Configuration/Block-Storage-RBD/rbd-mirroring.md

```diff
@@ -165,13 +165,13 @@ APIs for storage disaster recovery. The extended APIs are provided via Custom
 Resource Definition(CRD). Create the VolumeReplication CRDs on all the peer clusters.
 
 ```console
-kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.3.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
-kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.3.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplicationsclasses.yaml
```

Comment:
```
kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplicationsclasses.yaml
error: unable to read URL "https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplicationsclasses.yaml", server reported 404 Not Found, status code=404
```

the correct URL is `kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml`